extension/js/common/ui/key-import-ui.ts

```diff
@@ -73,15 +74,36 @@ export class KeyImportUi {
       } else if ((this as HTMLInputElement).value === 'backup') {
         window.location.href = Url.create('/chrome/settings/setup.htm', { acctEmail, parentTabId, action: 'add_key' });
       }
+      if (view !== undefined && view.submitKeyForAddrs.length > 1) {
+        $('.container_for_import_key_email_alias').css('visibility', 'visible');
+      }
     });
     $('.line.unprotected_key_create_pass_phrase .action_use_random_pass_phrase').click(Ui.event.handle(() => {
       $('.source_paste_container .input_passphrase').val(PgpPwd.random()).keyup();
       $('.input_passphrase').attr('type', 'text');
       $('#e_rememberPassphrase').prop('checked', true);
     }));
+    $('.input_private_key').on('keyup paste change', Ui.event.handle(async target => {
+      $('.action_add_private_key').addClass('btn_disabled').attr('disabled');
+      $('.input_email_alias').prop('checked', false);
+      const { keys: [prv] } = await opgp.key.readArmored(String($(target).val()));
+      if (prv !== undefined) {
+        $('.action_add_private_key').removeClass('btn_disabled').removeAttr('disabled');
+        if (view !== undefined && view.submitKeyForAddrs.length > 1) {
```

Comment:
we need to write to `view.submitKeyForAddrs` the users we check (if not already there)
and possibly uncheck (and remove from `view.submitKeyForAddrs`).
I see now what's possibly confusing you -- in `setup-render.ts` `renderInitial()` function `submitKeyForAddrs` is filled with all the aliases to pass to `renderEmailAddresses()` -- this should be refactored.
submitKeyForAddrs should be initially empty (`[]`) -- it would be checked based on key UIDs. But `renderEmailAddresses` should have a parameter `addresses` to render unchecked options.

And be called like: 
```
this.renderEmailAddresses(this.filterAddressesForSubmittingKeys(addresses))
```
and the function would start like
```
  private renderEmailAddresses = (addresses: string[]) => {
    $('.input_submit_all').hide();
    const emailAliases = Value.arr.withoutVal(addresses, this.view.acctEmail);
```